### DISCUSSION TITLE
예) 아이템 1을 읽으면서...

### DISCUSSION THREAD

**Discussion by Jaewui** [In-houseStudyGroup/Effective-Java-Study#6] [3. Q&A]
아이템 1을 읽으면서 궁금한 점이 있었는데, 

황성태 주임은 왜 이렇게 잘생겼죠?

월요일에 꼭 답변 부탁드립니다.

**Comment by 4862abd** [comment]
아이템 1을 읽으면서 저도 질문이 생겼습니다.
황성태 주임은 왜 외근을 가는거죠?
항상 옆에 붙여두고 싶은데..

**Reply by tjdxo1193** [reply]
ㅋㅋㅋㅋㅋㅋ
박병우 주임도 저랑 같이 가면 됩니다
